From d7f210e85c2e74124210a92628a9578d04eae187 Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Wed, 18 Sep 2024 22:48:55 -0400
Subject: [PATCH 5/5] Disable CMake's built-in packaging

It fails on Fedora 40 and we will just use our own RPM spec.

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 850edaa..326dced 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,7 +52,7 @@ add_subdirectory(third_party)
 add_subdirectory(umd)
 add_subdirectory(validation)
 
-include(cmake/packaging/setup.cmake)
+#include(cmake/packaging/setup.cmake)
 
 # You may your custom commands in .local.cmake, i.e. driver compilation, running tests.
 include(${CMAKE_CURRENT_SOURCE_DIR}/.local.cmake OPTIONAL)
-- 
2.46.1

